packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -79,35 +81,119 @@ type PrimitiveNavProps = Radix.ComponentPropsWithoutRef<typeof Primitive.nav>;
 interface NavigationMenuProps
   extends Omit<NavigationMenuProviderProps, keyof NavigationMenuProviderPrivateProps>,
     Omit<PrimitiveNavProps, 'defaultValue'> {
+  value?: string;
+  defaultValue?: string;
+  onValueChange?: (value: string) => void;
   dir?: Direction;
   orientation?: Orientation;
+  /**
+   * The duration from when the pointer enters the trigger until the tooltip gets opened.
+   * @defaultValue 0
+   */
+  delayDuration?: number;
+  /**
+   * How much time a user has to enter another trigger without incurring a delay again.
+   * @defaultValue 0
+   */
+  skipDelayDuration?: number;
 }
 
 const NavigationMenu = React.forwardRef<NavigationMenuElement, NavigationMenuProps>(
   (props: ScopedProps<NavigationMenuProps>, forwardedRef) => {
     const {
       __scopeNavigationMenu,
-      value,
+      value: valueProp,
       onValueChange,
       defaultValue,
+      delayDuration = 0,
+      skipDelayDuration = 0,
```

Comment:
Avoiding breaking change by not disturbing the current behaviour

Reply:
I'm not sure it would be considered a breaking change, I'd rather we start with good defaults and put a note in the changelog.

Reply:
That's true, especially if these defaults are blanket improvements.